### PR TITLE
Change the data type in the text to match the example

### PR DESCRIPTION
**Pull request by DemX86**
Problem: There is a `list` data type in the description text but the following example has the type of a `set`

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

**Review by parlough — APPROVED** [review]
Nice catch, looks good to me! Thanks :)

File: src/_guides/language/coming-from/js-to-dart.md [hunk 1]
```diff
@@ -1585,7 +1585,7 @@ Pure JavaScript doesn't support immutability.
 Dart offers multiple ways to make collections like arrays, sets, or
 dictionaries immutable.
 
-* If the list is a compile-time constant and shouldn't
+* If the set is a compile-time constant and shouldn't
```
Comment (parlough) [inline src/_guides/language/coming-from/js-to-dart.md:1588]: Actually this section might be better served with a more generic word like collection:

```suggestion
* If the collection is a compile-time constant and shouldn't
```

Comment (DemX86) [reply src/_guides/language/coming-from/js-to-dart.md:1588]: Agree, that will be better — thanks for the correction!

**Comment by parlough** [comment]
Looks good to me :)

**Pull request merged**